documentation/reference/charm.md

```diff
@@ -0,0 +1,46 @@
+# Charm.py Reference Documentation
+
+This file is the entrypoint for the charm, and contains functions for its basic operation, including its major hooks and file management. This file can be found at [src/charm.py](../../../src/charm.py).
```

Comment:
The link didn't work for me. Does it happen to you? Ah, maybe it's because of the PR context, so it would work when merged to main, I don't know.

Reply:
It works fine in VSCode, but you may be right about it not working in the PR. Happy to test it on main when it's merged in, or I can set it to a github link in main if you want.

Reply:
I think we can keep it the way it is. So we can confirm after it's merged.